471 docs   création dune histoire (#474)
JeremieLeymarie/story-builder: documentation/docs/Builder/builder_engine.mdx

You need to double click on the first scene card right in the middle so you can initialize your first scene.

Once you have a double clicked on it, a popup window will appear.

<img src="/img/builder_first_scene.png" alt="logo" width="1000" />

This "Edit scene" popup contains 2 main tabs named Scene and Actions. You need next to type a title and a description for your story.

The title is the name you give to your scene, you will be able to name every single scene you have created on Story Builder. This will allow you to keep tabs more

> easily on your story. For clarity purposes, we recommend that you use a coherent naming convention, so that you do not get lost while creating your story.

Reviewer: exigeant ahah

> The "first scene" allows you to turn any scene in the builder into the first "startign scene". Here again we advise you to be logical when choosing you first because it is esssentially the starting point of your story.

Reviewer: ```suggestion
The "first scene" allows you to turn any scene in the builder into the first "starting scene". Here again we advise you to be logical when choosing you first because it is essentially the starting point of your story.
```

> Content is the core narrative element of you scene. You will be able to write a short description initializing your scene and it will allow you to set the mood for your story.

Reviewer: ça c'est pas vraiment vrai, genre il faut être clair que le contenu c'est tout le texte d'une scène, c'est pas vraiment une _description courte_, et ça set pas forcément le mood de l'histoire, c'est vrai comme le contenu d'une page d'un livre 

pense d'un point de vue joueur·euse

Reviewer: ```suggestion
Content is the core narrative element of your scene. This is what the player/reader will see when reading your story. You can think of it as the content of page in a book. 
```

You can add images to your content description, if you click on the "image" button. Then you can copy paste an image url from the internet.

<img src="/img/builder_content_image_form.png" alt="logo" width="500" />

<img src="/img/builder_content_image.png" alt="logo" width="500" />

> Rich text is available to you in the description field. As you can see i used bold and itallic for my description text.

Reviewer: ```suggestion
Rich text is available to you in the description field. As you can see i used bold and italic for my description text.
```

You can access the "Actions" tab by clicking on it.

This is what you get once you have filled out all the necessary information:

<img src="/img/builder_first_scene2.png" alt="logo" width="1000" />

To create a new scene linked to one of the two actions you have to click on the black dot next to the chosen action.

<img src="/img/builder_new_scene.png" alt="logo" width="400" />

From then you just have to fill the scene like we just did to create your second scene.

> Actions are absolutely necessary if you want your story to be dynamic and have multiple choices for the player.

Reviewer: J'aurais dit qqchose dans ce goût là moi =>

```suggestion
_Actions_ represent the choices made by the player/reader throughout the story. They allow the reader to advance in the story. In the game, they are rendered as buttons that lead to the next page. 
```

Reviewer: jsp ce que t'en penses

Here if you click on the new empty scene and select the "Actions" tab, you can add new actions fields by clicking on the little "+" sign.

<img src="/img/builder_scene_actions.png" alt="logo" width="700" />

Here is a simple rule for you : More actions = More narrative possibilities for your story.

This is the very essence of Story Builder! Connecting various scenes together to create dynamic stories that are connected to each other.

> This allows you to create stories with multiple possibilities.

Reviewer: c'est peut-être un peu répétitif

```suggestion
```

<img src="/img/builder_first_scene.png" alt="logo" width="1000" />

This "Edit scene" popup contains 2 main tabs named Scene and Actions. You need next to type a title and a description for your story.

> This what you get once you have filled out all the necessary information :

Reviewer: ```suggestion
This is what you get once you have filled out all the necessary information:
```

<img src="/img/builder_first_scene2.png" alt="logo" width="1000" />

> Rich text is available to you in the description field. As you can see i used bold and itallic for my description text. You can access the "Actions" tab by clickign on it

Reviewer: ```suggestion
Rich text is available to you in the description field. As you can see i used bold and itallic for my description text. You can access the "Actions" tab by clicking on it
```

> To create a new scene linked to one of the two actions you have to click on the black dot next to the chosen action.

Reviewer: Je pense qu'on a envie d'avoir une description un peu plus haut niveau de ce que sont chacun des champs : titre, contenu et actions
genre expliquer à quoi ça correspond pour les joueur·euses, dans le jeu

<img src="/img/builder_new_scene.png" alt="logo" width="400" />

From then you just have to fill the scene like we just did to create your second scene.

> This is the very essence of Story Builder ! Connecting various scenes together to create dyanmic stories that are connected to each other.

Reviewer: ```suggestion
This is the very essence of Story Builder! Connecting various scenes together to create dynamic stories that are connected to each other.
```
